.github/workflows/deploy.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy to Heroku
+
+on:
+  push:
```

Comment:
I don't believe we will need on push because we should not push code to main without a PR.

Reply:
I believe this is just the syntax used to trigger it when we merge the code to main, which is the behavior we want. See top answer [here](https://stackoverflow.com/questions/60710209/trigger-github-actions-only-when-pr-is-merged).